src/SprykerEco/Client/Payone/PayoneClient.php

```diff
@@ -161,8 +161,7 @@ public function getInvoice(PayoneGetInvoiceTransfer $getInvoiceTransfer)
     }
 
     /**
-     * Send genericpayment request to payone with action "setexpresscheckout"
```

Comment:
Copy this description to the interface.

Reply:
The one in ClientInterface is already more general, so I've decided to skip it.